src/guide/installation.md

```diff
@@ -71,3 +76,68 @@ $ cd <project-name>
 $ yarn
 $ yarn dev
 ```
+
+## Explanation of Different Builds
+
+In the [`dist/` directory of the NPM package](https://cdn.jsdelivr.net/npm/vue@3.0.0-rc.1/dist/) you will find many different builds of Vue.js. Here is an overview of which `dist` file should be used depending on the use-case.
+
+### From CDN or without a Bundler
+
+#### `vue(.runtime).global(.prod).js`:
+
+- For direct use via `<script src="...">` in the browser. Exposes the Vue global.
```

Comment:
These two sentences are fragments, might be more welcoming if they become one sentence

Reply:
Thank you! Fixed 👍🏻